src/pages/doi.org/[...doi].tsx

```diff
@@ -507,38 +507,42 @@ const WorkPage: React.FunctionComponent<Props> = ({ doi, metadata, isBot = false
   const [registrationAgency] = useQueryState('registration-agency', { history: 'push' })
   const [connectionType] = useQueryState('connection-type', { history: 'push' })
 
-  const { loading, error, data } = useQuery<WorkQueryData, QueryVar>(isBot ? LIGHTWEIGHT_DOI_GQL : DOI_GQL, {
+  const variables = {
+    id: doi,
+    cursor: cursor,
+    filterQuery: filterQuery,
+    published: published as string,
+    resourceTypeId: resourceType as string,
+    fieldOfScience: fieldOfScience as string,
+    language: language as string,
+    license: license as string,
+    registrationAgency: registrationAgency as string
+  }
+
+  const doiQuery = useQuery<WorkQueryData, QueryVar>(DOI_GQL, { errorPolicy: 'all', variables: variables })
+  const relatedContentQuery = useQuery<WorkQueryData, QueryVar>(RELATED_CONTENT_GQL, {
     errorPolicy: 'all',
-    variables: {
-      id: doi,
-      cursor: cursor,
-      filterQuery: filterQuery,
-      published: published as string,
-      resourceTypeId: resourceType as string,
-      fieldOfScience: fieldOfScience as string,
-      language: language as string,
-      license: license as string,
-      registrationAgency: registrationAgency as string
-    }
+    variables: variables,
+    skip: isBot
```

Comment:
@jrhoads , here is an example of the isBot check

Reply:
Excellent.  Sorry I had missed it.

What happens to `loading`, `error` and `data` if the query is skipped?

Reply:
Loading is set to false, error and data are set to undefined. I have a check in the relatedContent components to early return if data isn't defined